appveyor.yml

```diff
@@ -11,7 +12,25 @@ install:
 
     python -m pip install --upgrade tox
 
+    SET PATH=%PYTHON64%;%PYTHON64%\\Scripts;%PATH%
+
+    python -m pip install --upgrade pip
+
+    python -m pip install --upgrade virtualenv
+
+    python -m pip install --upgrade tox
+
     mkdir C:\\tmp\\Trash
+
 build: off
 test_script:
-- cmd: tox -e py-ci-cov-codecov
+- cmd: >-
+    SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
+
+    tox -e py-ci-cov-codecov
+
+    if %errorlevel% neq 0 exit /b %errorlevel%
+
+    SET PATH=%PYTHON64%;%PYTHON64%\\Scripts;%PATH%
+
+    tox -e py-ci-cov-codecov

```

Comment:
Run without codecov in 64bit mode. This is not working and I don't think we need it.

Reply:
so tox -e py ?